### ISSUE TITLE
Install the RzComDriver without Synapse

### ISSUE THREAD

**Issue by Chaoses-Ib** [Chaoses-Ib/IbInputSimulator#7]
I also found that you can actually install the RzComDriver without Synapse. There are various logs generated by razer synapse and one of them is `Manifest.log` this contains an XML manifest of all software products from razer. I have a console app I wrote in c# that actually uses the C# API used by Razers software to download the same Manifest file.

Within the Manifest, there is a link to the full Razer Synapse install (eg: https://cdn.razersynapse.com/1665042231FcnE6gEARazerSynapseSetup_v3.7.1015.100615.exe).

This exe is in a format that can be extracted from and within the $PLUGINSDIR `RzComDriver_vx.x.xx.x.exe` exists which is the Razer Common Control Driver.

![image](https://user-images.githubusercontent.com/633360/196014549-d56f9e55-c26f-45db-8d5a-708fecc088e6.png)

_Originally posted by @ChrisKader in https://github.com/Chaoses-Ib/IbInputSimulator/issues/6#issuecomment-1279871932_

**Comment by Chaoses-Ib** [comment]
This is an amazing discovery. I will test it to see if it works properly.

**Comment by kyloren23** [comment]
sorry how do i use this? i tried it to my other laptop without razer and i install this v3.3.0331.041818
https://cdn.razersynapse.com/1524047122rATOSf1dRazerSynapseSetup_v3.3.0331.041818.exe 

but gave me error when i run ahk. something like "razer device is not found"

**Comment by Chaoses-Ib** [comment]
> i tried it to my other laptop without razer and i install this v3.3.0331.041818

The driver is installed by Nanoleaf, not Razer Synapse.

**Comment by ZhangWebber** [comment]
@Chaoses-Ib , How to install these?
I have installed but nothing is opening

**Comment by Chaoses-Ib** [comment]
Yeah, there is no GUI so you won't see anything. Just try to use IbInputSimulator to use the Razer driver.

**Comment by re-learning** [comment]
I've downloaded the razer synapse 3.7, extracted and installed RzComDriver. However nothing seems to work when i try to move mouse. I dont have a razer device. Do I need an older version?

**Comment by Chaoses-Ib** [comment]
Have you tried to install Nanoleaf listed avobe?

**Comment by re-learning** [comment]
I will try it tomorrow and report back

**Comment by re-learning** [comment]
Installing nanoleaf directly without extracting did work, thanks

**Comment by JQK6755576** [comment]
v3.5.0331.032014 (2020-03-20)
https://cdn.razersynapse.com/1584687554h47EvtnSRazerNanoleafSetup_v3.5.0331.032014.exe
v3.5.1030.101616 (2020-10-16)
https://cdn.razersynapse.com/16028372057849nNqJRazerNanoleafSetup_v3.5.1030.101616.exe 这个源码是否可分享
